src/app/shared/profile-header/profile-header.ts

```diff
@@ -112,6 +114,15 @@ export class ProfileHeaderComponent {
             scale: 5,
           });
         }
+        if(profile.nip05) {
+          let nip05Data = null
+          try {
+            nip05Data = await nip05.queryProfile(profile?.nip05)
```

Comment:
Didn't know nostr-tools had this, great!

Plan was to use a Web Worker to get and verify NIP-05s in the background, but this is likely fine. I will add saving of the result so it won't have to be checked every load.